sql/compact/src/main/java/NoClassExample.java

```diff
@@ -0,0 +1,55 @@
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.IMap;
+import com.hazelcast.nio.serialization.GenericRecord;
+import com.hazelcast.sql.SqlResult;
+import com.hazelcast.sql.SqlRow;
+import com.hazelcast.sql.SqlService;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Demonstrates usage of Compact(BETA) format with SQL Service without an existing DTO class
+ */
+public class NoClassExample {
+
+    public static void main(String[] args) throws InterruptedException {
+        Config config = new Config();
+        config.getJetConfig().setEnabled(true);
+        config.getSerializationConfig().getCompactSerializationConfig().setEnabled(true);
+        HazelcastInstance hazelcast = Hazelcast.newHazelcastInstance(config);
+        IMap<Integer, GenericRecord> myMap = hazelcast.getMap("myMap");
+
+        SqlService sqlService = hazelcast.getSql();
+
+        sqlService.execute("CREATE MAPPING myMap ( " +
```

Comment:
Can we add a comment on what this statement does at the background briefly. "It creates a table named myMap with columns listed, etc." Indicate the meaning of the `valueCompactTypeName ` and `valueFormat ` fields.